app/src/main/java/org/island/messaging/MessageLayer.java

```diff
@@ -137,7 +137,7 @@ private static void addFriendToDatabaseAndCreateDefaultProfile(Context context,
         }
     }
 
-    public static String getQrCode(Context context) {
+    public static String getEncodedString(Context context) {
```

Comment:
I think `getEncodedString` might be too general. Maybe something like `getMyEncodedIdentity` or `getEncodedIdentityString`?